packages/ember-runtime/tests/suites/mutable_array/insertAt.js

```diff
@@ -1,18 +1,19 @@
 import { SuiteModuleBuilder } from '../suite';
 import { get } from 'ember-metal';
+import { insertAt } from '../../../mixins/mutable_array';
 
 const suite = SuiteModuleBuilder.create();
 
 suite.module('insertAt');
 
-suite.test('[].insertAt(0, X) => [X] + notify', function() {
+suite.test('insertAt([], 0, X) => [X] + notify', function() {
```

Comment:
I think its beneficial to have some tests still use the array prototype until it is fully removed. I'm not sure if we have tests that specific check for this already elsewhere or if we should add those checks here

Reply:
I followed the example of https://github.com/emberjs/ember.js/pull/13534/files#diff-d1dc85945ef368d3bcdd3e01234d3f8eR118 and duplicated the last test, keeping it to use the array prototype extension. See https://github.com/emberjs/ember.js/pull/16048/files/ac0a1455b73d59a5b1256f81096343a90f846ebd#diff-eaa813784954ae3195242cb1d1763b7dR171

I agree it's not very clear though. I can change the test's label but not sure if the test itself needs to be more specific.

Reply:
I guess what I'm trying to get at is that we should maintain at least one test that simply checks that the prototypes are still extended. This could simply be something like:

```js
test('that prototypes are properly extended', function(assert) {
  assert.equal(typeof [].insertAt, 'function');
  // ... all other extensions
});
```

I don't know if we have something like this already but if not we should add it

Reply:
Do you mean something like https://github.com/emberjs/ember.js/blob/master/packages/ember-runtime/tests/system/string/camelize_test.js#L6-L10 but the inverse?

Added a commit for that for both `removeAt` and `insertAt`

Reply:
Yep that looks good. Thanks @wongpeiyi